examples/compute/main.c

```diff
@@ -0,0 +1,180 @@
+#include "./../../ffi/wgpu.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+#define BINDINGS_LENGTH (1)
+#define BIND_GROUP_LAYOUTS_LENGTH (1)
+
+WGPUByteArray read_file(const char *name) {
```

Comment:
could we move common stuff like this into a header shared between examples?

Reply:
Yes totally agreed, will add this to my to-do for a follow-up issue. Will do this and creating a shared cmake solution for all the examples, so we won't have to copy paste the CMakeLists.txt every time a new example is created.